lib/inspec/resources/postgres_hba_conf.rb

```diff
@@ -5,6 +5,7 @@ module Inspec::Resources
   class PostgresHbaConf < Inspec.resource(1)
     name "postgres_hba_conf"
     supports platform: "unix"
+    supports platform: "windows"
```

Comment:
I would assume we want to add accessors on all the resources, like, `attr_reader :params, :conf_file` etc.

Reply:
I did for the required one.